### PR TITLE
Fix:(initialize) remove fix_seed for initialization

### PR DESCRIPTION
**Pull request by li126com**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

https://github.com/InternLM/InternEvo/issues/18

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: internlm/model/modeling_internlm.py [hunk 1]
```diff
@@ -451,7 +438,6 @@ def _build_generic_model_1d(num_layers, num_chunks, device=torch.device("cuda"),
         logger.info(f"The layer sharding is {all_parts}.")
 
     models = []
-    PackedFlashInternLm1D.__init__ = fix_seed(PackedFlashInternLm1D.__init__)
 
     for start, end in parts:
         kwargs["num_layers"] = end - start
```
Comment (blankde) [inline internlm/model/modeling_internlm.py:443]: 佳兴，把fix_seed的逻辑给移除会导致这里的每一个chunks的模型用的是不一样的随机化状态初始化的，和这个PR https://github.com/InternLM/InternLM/pull/496 冲突吗

Comment (blankde) [reply internlm/model/modeling_internlm.py:443]: 已经和佳兴聊过不冲突

**Review by blankde — APPROVED** [review]

**Comment by sunpengsdu** [comment]
LGTM

**Review by sunpengsdu — APPROVED** [review]

**Comment by sunpengsdu** [comment]
@kkscilife This pull request for https://github.com/InternLM/InternEvo/issues/18

**Pull request merged**